recipes/bioconductor-deseq2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: bioconductor-deseq2
+  version: 1.12.4
+source:
+  fn: DESeq2_1.12.4.tar.gz
+  url: http://bioconductor.org/packages/release/bioc/src/contrib/DESeq2_1.12.4.tar.gz
+  md5: 08f115d77d4abe94f655be6c9f3144f8
+build:
+  number: 1
```

Comment:
This seems to be a new recipe and it starts with 1. Maybe a bug in the skeleton generator?
Will fix it.